docker-compose/example.env

```diff
@@ -24,6 +24,9 @@ RAY_MEMORY=100G
 FRONTEND_PROTOCOL=http
 SECURED_COOKIES="False"
 
+# Path to the local host file system where the Local File System connector will write the data
+SYNTHO_LOCALFS=/tmp/
```

Comment:
Can you also add this env in this .tpl file [here](https://github.com/syntho-ai/deployment-tools/blob/57571b5545df94e2c6d443583579d58823f5e7c2/docker-compose/.env.tpl#L26)? syntho-cli is using this .tpl when generating the real .env file when deploying. We actually have a PR checks to avoid this discrepancy but apparently we missed adding this check for docker compose, and added for helm charts only. We talked with @Younday and that PR check will be added soon for docker compose as well.

Reply:
It's maybe also a good thing to check whether we want to merge this in, I believe it was a temporary fix. I will however create another PR with a check in a pipeline to see if the keys in `.env.tpl` and `example.env`.